### DISCUSSION TITLE
Xamarin.Android.GridLayout renderers

### DISCUSSION THREAD

**Discussion by Jakar510** [xamarin/Xamarin.Forms#13899] [Q&A]
When the grid is rendered, the rows and columns are can be fixed sizes, relative or dynamic. I want them to be a constant relative size, to get a specific layout in native renderers, similar to how Xamarin.Forms (XF) GridLayout works, such as the following:

	<Grid.ColumnDefinitions>
			<ColumnDefinition Width="0.3*"/>
			<ColumnDefinition Width="0.5*"/>
			<ColumnDefinition Width="*"/>
	</Grid.ColumnDefinitions>

So basically, how are row and column definitions rendered in XF, or how do I fix the sizing?
